docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -281,7 +281,7 @@ Consult the [channel events section of the Channel Webhooks Guide](/api-docs/cha
 |:-------------------------|:------------|
 | store/customer/* | Fires for all store/customer events. |
 | store/customer/created | Fires when a new customer is created.|
-| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. |
+| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. Tracks changes to customer attributes only if the changes are made using the Control Panel. |
```

Comment:
One thing I thought worth mentioning is that updating customer attributes in CP would only trigger the same event (same type and same payload) as updating the customer. So, the information such as what customer attributes have been changed to would **not** show in the event.

Reply:
👍 Thanks, I updated the descriptions based on your feedback